blockchain/state_manager/Cargo.toml

```diff
@@ -10,9 +10,9 @@ features = ["forest_statediff"]
 [dependencies]
 anyhow                 = "1.0"
 async-log              = "2.0.0"
-async-std              = { version = "1.9", features = ["unstable", "tokio1"] }
-blake2b_simd           = "0.5.9"
-byteorder              = "1.3.4"
+async-std              = { version = "1.12", features = ["unstable", "tokio1"] }
+blake2b_simd           = "1.0.0"
+byteorder              = "1.4.3"
 cid                    = { version = "0.8", default-features = false, features = ["std"] }
 fil_actors_runtime     = "=8.0.0"
 filecoin-proofs-api    = { version = "11.0.0", default_features = false }
```

Comment:
Can we upgrade to `filecoin-proofs-api-12.0`?

Reply:
No, due to `blst` version conflict